src/component/reducer/helper/GetClosestNumber.ts

```diff
@@ -1,6 +1,7 @@
-function getClosestNumber(array: number[] = [], goal = 0) {
-  const closest = array.reduce((prev, curr) => {
-    return Math.abs(curr - goal) < Math.abs(prev - goal) ? curr : prev;
+function getClosestNumber(array: number[], goal = 0) {
+  let closest = Infinity;
+  array.forEach((curr) => {
+    closest = Math.abs(curr - goal) < Math.abs(closest - goal) ? curr : closest;
```

Comment:
I find this was a valid usage of `reduce` as it really "reduces" the array to one number